core/cap-0035.md

```diff
@@ -159,7 +142,22 @@ struct Operation
  
     body;
 };
-``` 
+```
+
+### Semantics
+
+#### Account
+This proposal requires the addition of `CLAWBACK_ENABLED` to the issuing account 
+AccountFlags so the XDR becomes

```

Comment:
Is the intentional to add the XDR here? Or should we just remove this line?

Reply:
Oops: e049165